src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -162,7 +161,7 @@
   (let [limit-clause     (if limit (format " LIMIT %s" limit) "")
         offset-clause    (if offset (format " OFFSET %s" offset) "")
         order-by-clause  (order-by->sql order_by)]
-    (format "SELECT paged_results.* FROM (%s) paged_results%s%s%s"
+    (format "SELECT paged_results.* FROM (%s %s%s%s) paged_results"
             sql order-by-clause limit-clause offset-clause)))
```

Comment:
Agreed... anywhere we're squishing together strings to form a query, we should think about using honeysql.